django-stubs/contrib/gis/utils/layermapping.pyi

```diff
@@ -2,8 +2,8 @@ from collections.abc import Mapping
 from pathlib import Path
 from typing import Any, Protocol
 
+import django.contrib.gis.gdal.field
 from django.contrib.gis.gdal import DataSource, OGRGeomType
-from django.contrib.gis.gdal.field import Field as OGRField
```

Comment:
`from django.contrib.gis.gdal.field import Field as OGRField` is preferrable here.

It doesn't count as a re-export; re-exports are only those that have the same name (e.g. `import Field as Field`)

Reply:
Documented here: https://typing.readthedocs.io/en/latest/source/stubs.html#imports

Reply:
Thank you for your review!